src/lib/validation/validation.ts

```diff
@@ -25,55 +21,60 @@ export default class Validation {
                 "suttonmail.org",
                 "highwaysengland.co.uk"
             ]
-    }
 
-    async loadExtendedEmailDomains(): Promise<void> {
         let extendedDomains: string[] = [];
-        await fs.readFile("./valid-email-domains.txt", "utf-8")
-            .then((emails) => {
-                    extendedDomains = emails.split("\n");
-                    this.validEmailDomains = this.validEmailDomains.concat(extendedDomains);
-                }
-            )
-            .catch(() => console.error("No extended email domains provided"))
+        try {
+            let emails = fs.readFileSync("./valid-email-domains.txt", "utf-8")
+            extendedDomains = emails.split("\n");
+            this.validEmailDomains = this.validEmailDomains.concat(extendedDomains);
+        } catch (error) {
+            console.error("No extended email domains provided")
+        }
```

Comment:
If the app can't read the config file on startup it should log and exit, and deployment shoud fail